src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -333,4 +333,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
     
   </Target>
+
+  <Target Name="CheckWindowsDesktopSdkInUse" 
+          BeforeTargets="_CheckForInvalidConfigurationAndPlatform" 
+          Condition="'$(UseWpf)' == 'true' Or '$(UseWindowsForms)' == 'true'">
+    <NetSdkWarning Condition="'$(_MicrosoftNetSdkWindowsDesktop)' != 'true'" 
```

Comment:
It looks like `_MicrosoftNetSdkWindowsDesktop` isn't set by the WindowsDesktop SDK yet.  Until it is, this will block WindowsDesktop projects from building.  I think you need to set that property in the WindowsDesktop SDK first, flow that through to core-sdk and then to stage 0 in this repo, before committing this change.

Reply:
I'm watching DARC flows (I triggered one earlier this morning from core-setup -> toolset exactly for this purpose: https://github.com/dotnet/toolset/pull/1177). Won't merge this until it's _very safe_ to do so.

Reply:
Update global.json with latest core-sdk that contains support for `_MicrosoftNetSdkWindowsDesktop`